File: 1031 - 1040/1036.py
A, B, C = map(float, input().split())

if A != 0:
    delta = (B * B) - (4 * A * C)

    if delta > 0:
        x1 = (-B + (delta**0.5)) / (2 * A)
        x2 = (-B - (delta**0.5)) / (2 * A)
        
        print("R1 = %.5f" % (x1))
        print("R2 = %.5f" % (x2))

    elif (delta < 0):
        print("Impossivel calcular")
else:
    print("Impossivel calcular")

File: 1031 - 1040/1036.c
#include<stdio.h>
#include<math.h>

int main(){
  double a,b,c;
  scanf("%le %le %le", &a, &b, &c);

  if (a != 0){
    double x1, x2;
    double delta;

    delta = pow(b, 2) - (4 * a * c);

    if (delta > 0){
        x1 = (-b + pow(delta, 0.5)) / (2 * a);
        x2 = (-b - pow(delta, 0.5)) / (2 * a);
        printf("R1 = %.5f\n", x1);
        printf("R2 = %.5f\n", x2);
    }

    else if (delta < 0){
      printf("Impossivel calcular\n");
    }

  }
  else {
    printf("Impossivel calcular\n");
  }

  return 0;
}
